### PR TITLE
Adding the capability of calling a Web API to the mvc2 template

### PR DESCRIPTION
**Pull request by jmprieur**
Adding the capability of calling a Web API to the mvc2 template

To test it:

```sh
msbuild /t:pack
cd bin\Debug
dotnet new -i Microsoft.Identity.Web.ProjectTemplates.0.1.4.nupkg
mkdir test
dotnet new mvc2 --auth SingleOrg --called-api-url "https://graph.microsoft.com/beta" --called-api-scopes "user.read"
```

to un-install the templates

```
cd bin\Debug
dotnet new -u Microsoft.Identity.Web.ProjectTemplates.0.1.4.nupkg
```

### CODE REVIEW

File: ProjectTemplates/templates/StarterWeb-CSharp/.template.config/dotnetcli.host.json [hunk 1]
```diff
@@ -69,9 +69,17 @@
       "shortName": ""
     },
     "RazorRuntimeCompilation": {
-      "longName": "razor-runtime-compilation",
-      "shortName": "rrc"
-    }
+        "longName": "razor-runtime-compilation",
```
Comment (jmprieur) [inline ProjectTemplates/templates/StarterWeb-CSharp/.template.config/dotnetcli.host.json:72]:
```suggestion
      "longName": "razor-runtime-compilation",
```

File: ProjectTemplates/templates/StarterWeb-CSharp/.template.config/dotnetcli.host.json [hunk 2]
```diff
@@ -70,8 +70,16 @@
     },
     "RazorRuntimeCompilation": {
       "longName": "razor-runtime-compilation",
-      "shortName": "rrc"
-    }
+        "shortName": "rrc"
```
Comment (jmprieur) [inline ProjectTemplates/templates/StarterWeb-CSharp/.template.config/dotnetcli.host.json:73]:
```suggestion
      "shortName": "rrc"
```

**Comment by jmprieur** [comment]
@jennyf19 : I've fixed the indentation issues.

**Comment by jennyf19** [comment]
@jmprieur looks great.
:shipit:

**Pull request merged**